packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -480,8 +486,9 @@ const New = decorate([
           },
         },
         settings: settings.set(id, {
-          exportRetention,
           copyRetention,
+          exportRetention,
+          fullInterval,
```

Comment:
I don't get this. If you enable the option on one schedule, then the `fullInterval` for the entire job will be set to 1?

Reply:
No, `setting.set(id, {}`  will set the schedule's settings corresponding to the passed `id`.